src/pybind/mgr/dashboard/frontend/src/app/core/auth/sso/sso-not-found/sso-not-found.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <div class="col-md-12 text-center">
+    <h1 i18n>Sorry, the user does not exist in Ceph.</h1>
```

Comment:
Thanks for taking care of this. One small suggestion (to avoid confusion):
```suggestion
    <h1 i18n>Sorry, this user account does not exist in Ceph Manager Dashboard.</h1>
```

Reply:
Done.